src/test/rustdoc-json/reexport_method_from_private_module.rs

```diff
@@ -0,0 +1,24 @@
+// Regression test for <https://github.com/rust-lang/rust/issues/102583>.
+
+// @has "$.index[*][?(@.name=='missing_from_rustdoc_json')]"
+// @!has "$.index[*][?(@.name=='hidden_impl')]"
+// @!has "$.index[*][?(@.name=='hidden_fn')]"
+
+#![no_std]
+
+mod private_mod {
+    pub struct S;
+
+    impl S {
+        pub fn missing_from_rustdoc_json() {}
```

Comment:
Nit: I don't think this name makes sense now that the issue is fixed.

Reply:
:+1: